app/controllers/idv/review_controller.rb

```diff
@@ -68,5 +76,13 @@ def submit_applicant
       idv_attempter.increment
       resolution
     end
+
+    def password
+      params.require(:user)[:password]
```

Comment:
I feel like I tried a half dozen different combinations to get what I wanted, which is:

* do not raise an exception if the param is missing
* do return nil if the param is missing

We want the error to come from the `valid_password?` method, not the controller param fetcher.